src/constants/local-storage.keys.ts

```diff
@@ -8,6 +8,9 @@ export default {
     TradeDeadline: 'app.tradeDeadline',
     EthereumTxType: 'app.ethereumTxType'
   },
+  Trade: {
+    Gasless: 'Trade.Gasless'
```

Comment:
Can we stick to the syntax used by all the other keys in this file (camelCase)?

Reply:
Also, can we now get rid of the `app.tradeInterface` key?

Reply:
yes! sorry my bad.